### DISCUSSION TITLE
Getting Invalid Key : unauthorized in my projectId

### DISCUSSION THREAD

**Discussion by dasSubhajit1999** [WalletConnect/walletconnect-monorepo#5103] [Developer Support]
I am using Wallet Connect v2 in my react application, I have generated a projectId but haven't added a domain, etc yet. This is showing me invalid key: unauthorized error. Is domain name verification necessary for making the wallet connect work in development mode

**Comment by glitch-txs** [comment]
> domain name verification necessary for making the wallet connect work in development mode

Hello, not at all. Could you share a screenshot or the full error log please?
